fetch.js

```diff
@@ -20,6 +20,10 @@ function WriteJsonFile(filename, jsonBody) {
 
 function TabifyJson(inputJson, filename, tabSpaces) {
     // console.log("  Begin tabifying: " + filename);
+
+    if (!inputJson)
+        return null;
```

Comment:
console.log("Empty inputJson, nothing to tabify"); ?

Reply:
The "rawResponse.length + bytes" comments, combined with the "Failed to GetApiFile" below make it sufficiently clear why it failed, so a comment here would be redundant.